### PR TITLE
Moved CVE reporting into its own runbit.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2932" title="DX-2932" target="_blank">TaskDX-2932</a>  Move CVE reporting out of the requirements runbit into its own runbit
  </td></table>
  <br />

### CODE REVIEW

**Comment by mitchell-as** [comment]
Failures are known and not introduced by this PR.

File: internal/runbits/requirements/requirements.go [hunk 1]
```diff
@@ -62,6 +55,10 @@ func (pv *PackageVersion) Set(arg string) error {
 }
 
 type RequirementOperation struct {
+	prime primeable
+	// The remainder is redundant with the above. Refactoring this will follow in a later story so as not to blow
+	// up the one that necessitates adding the primer at this level.
+	// https://activestatef.atlassian.net/browse/DX-2869
```
Comment (mitchell-as) [inline internal/runbits/requirements/requirements.go:61]: Copied from your PR to hopefully avoid a merge conflict.

File: internal/runbits/requirements/requirements.go [hunk 2]
```diff
@@ -558,99 +555,6 @@ func (r *RequirementOperation) resolveRequirement(requirement *Requirement) erro
...
-	}
-
-	names := requirementNames(requirements...)
-	pg := output.StartSpinner(r.Output, locale.Tr("progress_cve_search", strings.Join(names, ", ")), constants.TerminalAnimationInterval)
-
-	var ingredients []*request.Ingredient
-	for _, requirement := range requirements {
-		if requirement.Operation == types.OperationRemoved {
-			continue
-		}
```
Comment (mitchell-as) [inline internal/runbits/requirements/requirements.go:574]: This is the only functional change in the PR. I removed the `requirements` argument and construct the list of names from the `artifactChangeset`. I also removed the top-level iteration over `requirements`, which is superfluous.

**Review by Naatan — CHANGES_REQUESTED** [review]
Looks good! But we're still passing prime members around as function parameters. I think it would be cleaner if these all shared a struct that has the prime recorded on it, just like our runners normally behave. 

A runbit really ought to just be a runner, but one that unlike regular runners can be imported by other runners. Point being the code itself should mostly follow the same pattern.

**Comment by mitchell-as** [comment]
Is this kind of what you were looking for?

**Comment by Naatan** [comment]
> Is this kind of what you were looking for? [36be5f3](https://github.com/ActiveState/cli/commit/36be5f3a038cb05f8a64963609e557826518a583)

Yep that's it. To be honest I'm not totally sold on the prime pattern, but as of right now this feels consistent with how we use it elsewhere.

**Review by Naatan — APPROVED** [review]

**Pull request merged**